.gitignore

```diff
@@ -20,3 +21,17 @@ package.json
 binding.gyp
 READMEFIRST
 npm-debug.log
+
+# lua files
+src/lua/mraaLUA_wrap.cxx
```

Comment:
building in tree is not recommended for cmake (or any other builds really tbh), any good reason why you can't just build somewhere outside of git or simply add an ignore for your build directory? In any case such stuff doesn't belong in commits adding Lua support so please remove.